### DISCUSSION TITLE
[Question][Hw5] Late Submission Policy

### DISCUSSION THREAD

**Discussion by pingpingy1** [prosyslab-classroom/cs424-program-reasoning#302] [Q&A]
If I make a late submission that (considering the late-submission penalty) results in a worse grade than before,
would I be allowed to revert to the previous submission on GradeScope?
I want to try some more modifications, but my current trend is that the more I change things about my code, the worse it gets.
So, I don't want to lose even more points than I already have.

Thank you.

**Comment by sujin0529** [comment]
Hi,

We are grading with the best score you have submitted.
This means that you could revert to the results you previously submitted.
Therefore, you don't have to worry about the penalty that occurs because of late submission.

Thanks.

**Comment by pingpingy1** [comment]
That's good to know. Thank you!
